deployment.properties

```diff
@@ -0,0 +1,20 @@
+#
+# GRAKN.AI - THE KNOWLEDGE GRAPH
+# Copyright (C) 2018 Grakn Labs Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+repo.github.organisation=graknlabs
+repo.github.repository=client-python
```

Comment:
As per my comment above, I don't think we need a new `deployment.properties` file just to contain the repository address. What do you think?

Reply:
@haikalpribadi `deployment.properties` is kind of standard way to pass properties. Do we want really want to refactor the rules and its usage across all `@graknlabs_bazel_distribution` to take in repo URLs as rule attributes?

Reply:
well, no, we don't want to refactor everything for this. But now we have 2 deployment.properties for every repo (one from build-tools and one from root). And it gets a bit confusing. Does one override the other? I imagine `deployment.properties` should contain "global" values that are shared across all repos. But _"where deploy_github should deploy to"_ feels like a one-off argument which is very event-specific, right?

Reply:
What if for `deploy_github` we pass `orgnisation` and `repository` arguments? Then we don't need to introduce a new `deployment.properties` file for each repo, right? Once there are more variables we need to manage per repo, then we refactor again to move them over to a `deployment.properties` file because then it would be worth managing them in a new file. What do you think?

Reply:
@haikalpribadi 
>But "where deploy_github should deploy to" feels like a one-off argument which is very event-specific, right?

Does it mean that we should **not** read values from `deployment.properties` for `deploy_github` and _always_ take in argument? We also use `deploy_github` in `grakn-core` where value is currently read from `build-tools`'s `deployment.properties`.